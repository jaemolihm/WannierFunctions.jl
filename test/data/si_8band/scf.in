 &control
    calculation     = 'scf'
    prefix          = 'si'
    restart_mode    = 'from_scratch'
    pseudo_dir      = '~/PP/all/'
    outdir          = './temp/'
    tprnfor         = .true.
    tstress         = .true.
 /
 &system
    ibrav     = 2
    celldm(1) = 10.26328
    nat       = 2
    ntyp      = 1
    ecutwfc   = 70.0
 /
 &electrons
    diagonalization = 'david'
    mixing_beta     = 0.7
    conv_thr        = 1.0d-15
 /
ATOMIC_SPECIES
 Si  28.0855  Si.dojo.sr.ONCVv0.4.lda.standard.upf
ATOMIC_POSITIONS crystal
 Si  0.125   0.125   0.125
 Si -0.125  -0.125  -0.125
K_POINTS automatic
 14 14 14 0 0 0

