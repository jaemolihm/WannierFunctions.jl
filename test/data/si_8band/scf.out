MPI startup(): Warning: I_MPI_PMI_LIBRARY will be ignored since the hydra process manager was found
MPI startup(): Warning: I_MPI_PMI_LIBRARY will be ignored since the hydra process manager was found
MPI startup(): Warning: I_MPI_PMI_LIBRARY will be ignored since the hydra process manager was found
MPI startup(): Warning: I_MPI_PMI_LIBRARY will be ignored since the hydra process manager was found

     Program PWSCF v.7.0 starts on 30Jul2022 at 16:13:49 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
         "P. Giannozzi et al., J. Chem. Phys. 152 154105 (2020);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI & OpenMP), running on       4 processor cores
     Number of MPI processes:                 4
     Threads/MPI process:                     1

     MPI processes distributed on     1 nodes
     674 MiB available memory on the printing compute node when the environment starts
 
     Reading input from scf.in

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  4
 
     K-points division:     npool     =       4
     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used

 
     G-vector sticks info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Sum        1027    1027    313                21547    21547    3527
 
     Using Slab Decomposition
 


     bravais-lattice index     =            2
     lattice parameter (alat)  =      10.2633  a.u.
     unit-cell volume          =     270.2704 (a.u.)^3
     number of atoms/cell      =            2
     number of atomic types    =            1
     number of electrons       =         8.00
     number of Kohn-Sham states=            4
     kinetic-energy cutoff     =      70.0000  Ry
     charge density cutoff     =     280.0000  Ry
     scf convergence threshold =      1.0E-15
     mixing beta               =       0.7000
     number of iterations used =            8  plain     mixing
     Exchange-correlation= SLA  PW   NOGX NOGC
                           (   1   4   0   0   0   0   0)

     celldm(1)=  10.263280  celldm(2)=   0.000000  celldm(3)=   0.000000
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (  -0.500000   0.000000   0.500000 )  
               a(2) = (   0.000000   0.500000   0.500000 )  
               a(3) = (  -0.500000   0.500000   0.000000 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = ( -1.000000 -1.000000  1.000000 )  
               b(2) = (  1.000000  1.000000  1.000000 )  
               b(3) = ( -1.000000  1.000000 -1.000000 )  


     PseudoPot. # 1 for Si read from file:
     ~/PP/all/Si.dojo.sr.ONCVv0.4.lda.standard.upf
     MD5 check sum: Not computed, couldn't open file
     Pseudo is Norm-conserving + core correction, Zval =  4.0
     Generated using ONCVPSP code by D. R. Hamann
     Using radial grid of 1510 points,  6 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1
                l(5) =   2
                l(6) =   2

     atomic species   valence    mass     pseudopotential
        Si             4.00    28.08550     Si( 1.00)

     48 Sym. Ops., with inversion, found (36 have fractional translation)



   Cartesian axes

     site n.     atom                  positions (alat units)
         1           Si  tau(   1) = (  -0.1250000   0.1250000   0.1250000  )
         2           Si  tau(   2) = (   0.1250000  -0.1250000  -0.1250000  )

     number of k points=   104

     Number of k-points >= 100: set verbosity='high' to print them.

     Dense  grid:    21547 G-vectors     FFT dimensions: (  40,  40,  40)

     Estimated max dynamical RAM per process >      21.23 MB

     Estimated total dynamical RAM >      69.13 MB

     Initial potential from superposition of free atoms

     starting charge       7.9989, renormalised to       8.0000
     Starting wfcs are    8 randomized atomic wfcs

     total cpu time spent up to now is        0.7 secs

     Self-consistent Calculation

     iteration #  1     ecut=    70.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  2.0

     Threshold (ethr) on eigenvalues was too large:
     Diagonalizing with lowered threshold

     Davidson diagonalization with overlap
     ethr =  5.90E-04,  avg # of iterations =  2.1

     total cpu time spent up to now is        2.2 secs

     total energy              =     -17.04753246 Ry
     estimated scf accuracy    <       0.05593372 Ry

     iteration #  2     ecut=    70.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  6.99E-04,  avg # of iterations =  1.0

     total cpu time spent up to now is        2.8 secs

     total energy              =     -17.04966063 Ry
     estimated scf accuracy    <       0.00247503 Ry

     iteration #  3     ecut=    70.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  3.09E-05,  avg # of iterations =  2.9

     total cpu time spent up to now is        3.5 secs

     total energy              =     -17.05047559 Ry
     estimated scf accuracy    <       0.00009053 Ry

     iteration #  4     ecut=    70.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.13E-06,  avg # of iterations =  4.3

     total cpu time spent up to now is        4.5 secs

     total energy              =     -17.05054667 Ry
     estimated scf accuracy    <       0.00000260 Ry

     iteration #  5     ecut=    70.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  3.25E-08,  avg # of iterations =  4.7

     total cpu time spent up to now is        5.5 secs

     total energy              =     -17.05054934 Ry
     estimated scf accuracy    <       0.00000016 Ry

     iteration #  6     ecut=    70.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  2.02E-09,  avg # of iterations =  3.3

     total cpu time spent up to now is        6.4 secs

     total energy              =     -17.05054940 Ry
     estimated scf accuracy    <       0.00000002 Ry

     iteration #  7     ecut=    70.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  2.90E-10,  avg # of iterations =  2.7

     total cpu time spent up to now is        7.1 secs

     total energy              =     -17.05054941 Ry
     estimated scf accuracy    <          5.7E-11 Ry

     iteration #  8     ecut=    70.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  7.07E-13,  avg # of iterations =  7.7

     total cpu time spent up to now is        8.5 secs

     total energy              =     -17.05054941 Ry
     estimated scf accuracy    <          3.2E-11 Ry

     iteration #  9     ecut=    70.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  4.04E-13,  avg # of iterations =  2.3

     total cpu time spent up to now is        9.3 secs

     total energy              =     -17.05054941 Ry
     estimated scf accuracy    <          3.9E-11 Ry

     iteration # 10     ecut=    70.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  4.04E-13,  avg # of iterations =  1.7

     total cpu time spent up to now is        9.9 secs

     total energy              =     -17.05054941 Ry
     estimated scf accuracy    <          6.1E-13 Ry

     iteration # 11     ecut=    70.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-13,  avg # of iterations =  1.0

     total cpu time spent up to now is       10.5 secs

     total energy              =     -17.05054941 Ry
     estimated scf accuracy    <          4.8E-13 Ry

     iteration # 12     ecut=    70.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-13,  avg # of iterations =  1.0

     total cpu time spent up to now is       11.1 secs

     total energy              =     -17.05054941 Ry
     estimated scf accuracy    <          7.8E-14 Ry

     iteration # 13     ecut=    70.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-13,  avg # of iterations =  1.0

     total cpu time spent up to now is       11.7 secs

     total energy              =     -17.05054941 Ry
     estimated scf accuracy    <          4.4E-14 Ry

     iteration # 14     ecut=    70.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-13,  avg # of iterations =  1.0

     total cpu time spent up to now is       12.3 secs

     End of self-consistent calculation

     Number of k-points >= 100: set verbosity='high' to print the bands.

     highest occupied level (ev):     6.0323

!    total energy              =     -17.05054941 Ry
     estimated scf accuracy    <          1.4E-16 Ry

     The total energy is the sum of the following terms:
     one-electron contribution =       4.84713717 Ry
     hartree contribution      =       1.09532349 Ry
     xc contribution           =      -6.19744983 Ry
     ewald contribution        =     -16.79556023 Ry

     convergence has been achieved in  14 iterations

     Forces acting on atoms (cartesian axes, Ry/au):

     atom    1 type  1   force =     0.00000000    0.00000000   -0.00000000
     atom    2 type  1   force =    -0.00000000   -0.00000000    0.00000000

     Total force =     0.000000     Total SCF correction =     0.000000


     Computing stress (Cartesian axis) and pressure

          total   stress  (Ry/bohr**3)                   (kbar)     P=      -18.85
  -0.00012815   0.00000000   0.00000000          -18.85        0.00        0.00
   0.00000000  -0.00012815  -0.00000000            0.00      -18.85       -0.00
   0.00000000  -0.00000000  -0.00012815            0.00       -0.00      -18.85


     Writing all to output data dir ./temp/si.save/
 
     init_run     :      0.48s CPU      0.49s WALL (       1 calls)
     electrons    :     11.54s CPU     11.61s WALL (       1 calls)
     forces       :      0.17s CPU      0.17s WALL (       1 calls)
     stress       :      0.35s CPU      0.35s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.39s CPU      0.39s WALL (       1 calls)
     potinit      :      0.02s CPU      0.02s WALL (       1 calls)
     hinit0       :      0.04s CPU      0.05s WALL (       1 calls)

     Called by electrons:
     c_bands      :     10.02s CPU     10.06s WALL (      15 calls)
     sum_band     :      1.36s CPU      1.37s WALL (      15 calls)
     v_of_rho     :      0.09s CPU      0.10s WALL (      15 calls)
     mix_rho      :      0.06s CPU      0.07s WALL (      15 calls)

     Called by c_bands:
     init_us_2    :      0.61s CPU      0.62s WALL (     858 calls)
     init_us_2:cp :      0.61s CPU      0.62s WALL (     858 calls)
     cegterg      :      6.58s CPU      6.62s WALL (     390 calls)

     Called by *egterg:
     cdiaghg      :      0.05s CPU      0.05s WALL (    1455 calls)
     h_psi        :      6.38s CPU      6.42s WALL (    1507 calls)
     g_psi        :      0.08s CPU      0.08s WALL (    1091 calls)

     Called by h_psi:
     h_psi:calbec :      0.23s CPU      0.23s WALL (    1507 calls)
     vloc_psi     :      5.91s CPU      5.94s WALL (    1507 calls)
     add_vuspsi   :      0.21s CPU      0.22s WALL (    1507 calls)

     General routines
     calbec       :      0.25s CPU      0.25s WALL (    1637 calls)
     fft          :      0.03s CPU      0.03s WALL (      53 calls)
     ffts         :      0.01s CPU      0.01s WALL (      15 calls)
     fftw         :      6.00s CPU      6.03s WALL (   11714 calls)
 
     Parallel routines
 
     PWSCF        :     12.62s CPU     13.03s WALL

 
   This run was terminated on:  16:14: 2  30Jul2022            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
